service/volume.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 // DeleteVolumes deletes the data volumes of service and its dependencies.
-func (s *Service) DeleteVolumes() error {
+func (s *Service) DeleteVolumes(c container.Container, ss *Service) error {
```

Comment:
we already have the service here `func (s *Service)`

Reply:
done